### ISSUE TITLE
Bots are close to be fixed ! 08/05/2016

### ISSUE THREAD

**Issue by LAWX** [davidmann4/ogar-feeder-bot#557]
# i have just find a solution so the bots connect to the server and they spawn !!!!
## Last thing to do it's to update the userscript  ! and fix the (Access-Control-Allow-Origin) error :+1:

Any new bots release will be uploaded to my channel 
**# Link : (https://www.youtube.com/user/3IGUP)**

**Comment by Nick8fr32** [comment]
can u pr ur fix

**Comment by LAWX** [comment]
i'm trying to find a solution here !!

**Comment by MasonBurdette** [comment]
The fix is easy

**Comment by Nick8fr32** [comment]
no its not @MasonBurdette XD

**Comment by MasonBurdette** [comment]
Just make a chrome extension that replaces agario with your own client

**Comment by Nick8fr32** [comment]
lol extensions dont work anymore??

**Comment by LAWX** [comment]
u can't just came and make a new extention with no working javascripts -_-

**Comment by MasonBurdette** [comment]
what do you mean? I use the chrome APIs to replace Agar.io html with Agar.io/v72 html. It's really not hard at all. ;)

**Comment by ghost** [comment]
we fixed this before agario client is broken now working on new one I did get spawing bots from yestday yes spawing is easy but client is broken need new one almost 25% of new client is done

**Comment by MasonBurdette** [comment]
Yes much of the protocol has changed. You can use something like `wireshark` to sniff the new packets. Then use `buffer-dataview` to replicate the packets. After that you can send them to the `agario server` using `websockets`.  Good Luck!

**Comment by ghost** [comment]
@MasonBurdette  extensions dont work lol

**Comment by MasonBurdette** [comment]
@problemsofworld you obviously don't understand then..  I have no clue what you are talking about.

**Comment by ghost** [comment]
agario extensions are down if you use any extension you cant join any party but still works with tampermokey I dont know if it works with you or not :/

**Comment by ghost** [comment]
all extensions are down  even agarplus :/

**Comment by LostCodesy** [comment]
@MasonBurdette give me your skype.

**Comment by LostCodesy** [comment]
@LAWX do are you fixed socket closed error?

**Comment by fayizan** [comment]
@MasonBurdette I understood about http://agar.io/v72/ and could you explain about wireshark My skype fayizbe

**Comment by ghost** [comment]
I tried my best it's imposseble to fix no hope even agar client maker said if you it's easy to fix then you are not real coder you dont understand I fucked spend 20Hours it's impossible to FIX protocol is changed no hope

**Comment by k69sknk** [comment]
Guys we need to create and organize a working group to fix that. #savebots

**Comment by ghost** [comment]
http://agar.io/v72/ is patched its no longer possible to fix it was the only hope

**Comment by luloak2** [comment]
@LAWX If you (or anyone else) needs some help deobfuscating the new protocol, let me know ;)

**Comment by ghost** [comment]
i need help deobfuscating the new protocol

**Comment by LAWX** [comment]
I'm trying so hard but there is no respond from the server ! :(

**Comment by luloak2** [comment]
@LAWX We should start a working group. Add me on skype if you want to, same username as I here.

**Comment by Andrews54757** [comment]
It wouldnt work anyway as you need to send a special key to the agario server that changes over time

**Comment by ImHITMan** [comment]
@Andrews54757 Is right, There is now a thing where You need a special key which grants access to agario servers, But the key changes everyday
